packages/frontend/app/components/learner-groups/root.hbs

```diff
@@ -122,7 +122,7 @@
           @setSortBy={{@setSortBy}}
         />
       {{else}}
-        <PulseLoader />
+        <LearnerGroups::Loading />
```

Comment:
This accepts a `count` of all the learner groups that will eventually show up on the page and has a row for each one. I think you'll need to massage the data loading a bit to get the same info you need here.

Reply:
I've tried a couple times to figure out how to load the count for Learner Groups, but it does it differently than Instructor Groups, as the model relationships are different. Can I get a hint? :P